tests/common/devices.py

```diff
@@ -268,6 +268,22 @@ def _get_platform_info(self):
                 result["hwsku"] = line.split(":")[1].strip()
             elif line.startswith("ASIC:"):
                 result["asic_type"] = line.split(":")[1].strip()
+
+        if result["platform"]:
+            platform_file_path = os.path.join('/usr/share/sonic/device', result["platform"], 'platform.json')
+
+            try:
+                out = self.command("cat {}".format(platform_file_path))
+                platform_info = json.loads(out["stdout"])
+                result["chassis"] = platform_info.get('chassis', None)
+                result["interfaces"] = platform_info.get('interfaces', None)
```

Comment:
Can we eliminate the need to know what keys are available? Optimally, we would like to insert all data from platform.json into `result`. Can you make this more generic?

Reply:
Thanks, converted this to a for loop.
Should be fixed